README.md

```diff
@@ -209,7 +209,7 @@ To display the points bar, add the following to the top of the assignment README
 If you want to float the points bar to the right, use the following instead:
 
 ```html
-<img alt="points bar" align="right" height="36" src="../../blob/status/.github/badges/points-bar.svg" />
+<img alt="points bar" align="right" height="36" src="../../blob/badges/.github/badges/points-bar.svg" />
```

Comment:
This was by far the biggest point of confusion for me.

Both previous code snippets were copy-pasteable, so everything would work "out-of-the-box". So I assumed the same for this `<img>` example, but it failed, and I had to go dig into URL to find out why.

The changes below were just to ensure that the same snippets will work for reusable workflow as well.